src/queries/vote/index.ts

```diff
@@ -165,3 +166,10 @@ export const useAdditionalInfoById = (id?: number) => {
 
   return items?.find(item => item.id === id);
 };
+
+export const usePlatformDayQuery = () => {
```

Comment:
What difference between platform day and loan platform day?

Reply:
I don't know if it's any different but this is a task request. Please check the attached image 
![ev1](https://user-images.githubusercontent.com/17588370/132805857-9d9714cb-9738-4531-831d-7c0d049adf17.png)

Reply:
You can use the regular platform day, that was a mistake I think